test/Rhetos.LightDMS.IntegrationTest/DownloadS3StorageTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Rhetos.LightDMS.IntegrationTest.Utilities;
+using Rhetos.LightDMS.TestApp;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Rhetos.LightDMS.IntegrationTest
+{
+    public class RhetosEntiy
+    {
+        public string ID { get; set; }
+    }
+
+    [Collection("S3 Storage simulator - local files")]
+    public class DownloadS3StorageTests : IDisposable
+    {
+        private static WebApplicationFactory<Startup> _factory;
+
+        private readonly Guid _documentVersionId = Guid.NewGuid();
+        private readonly string _fileContent = "Test file content";
+        private readonly Guid _fileContentId = Guid.Parse("b83930a2-c8ab-4678-8457-e55bdcb18aa5");

```

Comment:
Why don't we use `Guid.NewGuid()` instead?

Reply:
Updated.